apps/kubernetes-dashboard/helm-release.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: kubernetes-dashboard
+spec:
+  interval: 1h
+  chart:
+    spec:
+      chart: kubernetes-dashboard
+      sourceRef:
+        kind: HelmRepository
+        name: kubernetes-dashboard
+      interval: 1h
+  values:
+    extraArgs:
+      - --enable-skip-login 
+      - --enable-insecure-login
+    protocolHttp: true
+    service:
+      externalPort: 80
+    ingress:
+      enabled: true
+      annotations:
+        cert-manager.io/cluster-issuer: letsencrypt-prod
+        traefik.ingress.kubernetes.io/router.entrypoints: websecure,web
+        traefik.ingress.kubernetes.io/router.tls: "true"
+      hosts:
+        - kubernetes.apps.chaosdorf.space
+      tls:
+        - secretName: netbox-tls
```

Comment:
don't think netbox has anything to do with this?

Reply:
good catch, thanks :)